eks/cdk/eks.py

```diff
@@ -64,7 +66,7 @@ def __init__(self, app: core.App, id: str, props, **kwargs) -> None:
 
         eks.add_helm_chart("cluster-autoscaler",
             chart = "cluster-autoscaler",
-            release = "cluster-autoscaler",
+            release = "aws-cluster-autoscaler",
```

Comment:
Change the name of the Helm chart release to be the same as the kubernetes deployment